cluster/src/main/scala/io/snappydata/impl/LeadImpl.scala

```diff
@@ -239,6 +240,22 @@ class LeadImpl extends ServerImpl with Lead
     }
   }
 
+  private def checkAuthProvider(props: Properties): Unit = {
+    var authP = props.getProperty(Attribute.AUTH_PROVIDER)
+    if (authP == null) {
+      authP = props.getProperty(Attribute.SERVER_AUTH_PROVIDER)
+    }
```

Comment:
what if Attribute.AUTH_PROVIDER and Attribute.SERVER_AUTH_PROVIDER are both non null ; Attribute.AUTH_PROVIDER is ldap and  Attribute.SERVER_AUTH_PROVIDER is not ldap.